frontend/src/app/accountCreation/MfaTotp/MfaTotp.tsx

```diff
@@ -1,15 +1,43 @@
+import { useEffect, useState } from "react";
+import { Redirect } from "react-router";
+
 import { Card } from "../../commonComponents/Card/Card";
 import { CardBackground } from "../../commonComponents/CardBackground/CardBackground";
 import Button from "../../commonComponents/Button/Button";
 import StepIndicator from "../../commonComponents/StepIndicator";
 import { accountCreationSteps } from "../../../config/constants";
+import iconLoader from "../../../../node_modules/uswds/dist/img/loader.svg";
+import { LoadingCard } from "../LoadingCard/LoadingCard";
 
 interface Props {
-  qrCode: string;
+  enrollFunction: Function;
```

Comment:
Generalized this component to be used both for Google Auth and Okta Verify - the API enrollment function is passed through as a prop

Reply:
Based on its use below, it seems like this could be typed as `() => Promise<{ qrcode: string }>`

Reply:
Fixed in c7c755e